packages/lib/turndownService.ts

```diff
@@ -2,11 +2,48 @@ import TurndownService from "turndown";
 
 const turndownService = new TurndownService();
 
-turndownService.addRule("newLine", {
-  filter: ["br"],
-  replacement: () => {
+function turndown(html: string | TurndownService.Node): string {
+  let result = turndownService.turndown(html);
+  result = result.replaceAll("[<p><br></p>]", "");
+
+  if (result === "<p><br></p>") {
+    result = "";
+  }
+
+  return result;
+}
+
+turndownService.addRule("shiftEnter", {
```

Comment:
Add different rules for shift-enter and normal enter. This fixes the thrown error and the goal is to that shift-enter and normal enter should look the same